src/js/dqa.jsx

```diff
@@ -80,20 +80,15 @@ var DQA = React.createClass({
     },
 
     _trimToLength: function(str, maxLengthInPixels) {
-        /*  Note: Always keep at least one word because column renderers of
-            fixed-data-table will always display at least part of the first word
-            (all words after the first are either displayed in full length or 
-            completely chopped off).
-        */
-        if (str.indexOf(' ') < 0) {
-            return str;
-        }
-
         var data = str.trim().split(' ');
-        var res = data[0];
-
-        for (var i=1; i<data.length; i++) {
-            if (this._getPixelLength(res + ' ' + data[i]) > maxLengthInPixels-20) {
+        var res = '';
+        maxLengthInPixels -= 20;
```

Comment:
Add comment about why 20 pixels are being subtracted, otherwise this is considered a *magic number*.